token/parse_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"go.step.sm/crypto/jose"
+	"gopkg.in/square/go-jose.v2/jwt"
```

Comment:
You can avoid this import using jose.Audience. `"go.step.sm/crypto/jose"` has aliases for almost everything we use.

Reply:
a5e944ec